trompace/client/client_itemlist.py

```diff
@@ -0,0 +1,128 @@
+
+from trompace.config import config
+from trompace.connection import submit_query
+from trompace.mutations.itemlist import *
+from trompace.constants import ItemListOrderType
+
+
+
+def create_itemlist_node(contributor = contributor, name = name, 
+                         description = description, ordered = ordered):
+    """
+    """
+    itemlistorder = ItemListOrderType.ItemListUnordered
+    if ordered:
+        itemlistorder = ItemListOrderType.ItemListOrderAscending
+
+    mutation = mutation_create_itemlist(contributor = contributor,
```

Comment:
take a look at the messages from the bot on this pull request - when you call methods with named parameters there are no spaces around equals: `contributor=contributor`